canvas_modules/harness/lib/application.js

```diff
@@ -14,32 +14,39 @@
  * limitations under the License.
  */
 
-"use strict";
 // ESLint Rule Overrides
 
 /* eslint no-process-exit: 0 */
-const express = require("express");
-const session = require("express-session");
-const compression = require("compression");
-const path = require("path");
-const appConfig = require("./utils/app-config");
-const constants = require("./constants");
-const log4js = require("log4js");
-const bodyParser = require("body-parser");
-const log4jsUtil = require("./utils/log4js-util");
-log4jsUtil.init();
+import express from "express";
+import session from "express-session";
+import compression from "compression";
+import path from "path";
+import { fileURLToPath } from "url";
+import appConfig from "./utils/app-config.js";
+import { APP_SESSION_KEY, API_PATH_V1, APP_PATH } from "./constants.js";
+import log4js from "log4js";
+import bodyParser from "body-parser";
+import log4jsUtils from "./utils/log4js-util.js";
+import webpack from "webpack";
```

Comment:
All the webpack imports should be dynamic imports and moved back to the hmr method since they aren't needed in production mode.  

For example
```
const webpack = await import("webpack");
```